mne/viz/_mpl_figure.py

```diff
@@ -1593,15 +1597,23 @@ def _draw_one_scalebar(self, x, y, ch_type):
         from .utils import _simplify_float
         color = '#AA3377'  # purple
         kwargs = dict(color=color, zorder=self.mne.zorder['scalebar'])
-        scaler = 1 if self.mne.butterfly else 2
-        inv_norm = (scaler *
-                    self.mne.scalings[ch_type] *
-                    self.mne.unit_scalings[ch_type] /
-                    self.mne.scale_factor)
+        
+        if ch_type == 'time':
+            label = f'{self.mne.boundary_times[1]/2:.2f} seconds'
+            text = self.mne.ax_main.text(x[1], y[1], label, va='bottom',
+                                         ha='right', size='xx-small', **kwargs)
```

Comment:
```suggestion
            text = self.mne.ax_main.text(x[1], y[1], label, va='bottom',
                                         ha='center', size='xx-small',
                                         **kwargs)
```
horizontal-alignment with "bottom" might look better here.

Reply:
I don't think there is a "bottom" option for horizontal alignment. I also prefer it with "right" horizontal-alignment, as it sits on top of the scale bar, whereas "center" horizontal alignment is has the text off-center from the scalebar.